scripts/docker/promote.sh

```diff
@@ -4,15 +4,15 @@ set -eo pipefail
 
 CLEAR='\033[0m'
 RED='\033[0;31m'
-PUBLISH=0
-GCR_REPO=gcr.io/o1labs-192920
+REGISTRY=""
 QUIET=""
 ARCH=amd64
 while [[ "$#" -gt 0 ]]; do case $1 in
   -n|--name) NAME="$2"; shift;;
   -v|--version) VERSION="$2"; shift;;
   -t|--tag) TAG="$2"; shift;;
-  -p|--publish) PUBLISH=1; ;;
+  -r|--registry) REGISTRY="$2"; shift;;
```

Comment:
I'd rename these args from `-r|--registry` -> `--pull-registry` and `--publish_registry` -> `--push-registry`. For clarify. Is that the intention?